<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaEscritorio</class>
 <widget class="QMainWindow" name="ventanaEscritorio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Escritorio remoto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="botonCapturar">
        <property name="text">
         <string>Capturar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonGuardar">
        <property name="text">
         <string>Guardar captura</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonPantallaCompleta">
        <property name="text">
         <string>Pantalla completa</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkTeclado">
        <property name="text">
         <string>Enviar teclado y rat√≥n</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkStreaming">
        <property name="text">
         <string>Streaming continuo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkGuardar">
        <property name="text">
         <string>Guardar automaticamente</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelCalidad">
        <property name="text">
         <string>Calidad: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderCalidad">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdCalidad">
        <property name="value" stdset="0">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progresoDescarga">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
